test_app/views.py

```diff
@@ -57,7 +58,7 @@ class TeamViewSet(TestAppViewSet):
 
 class UserViewSet(DABOAuth2UserViewsetMixin, TestAppViewSet):
     queryset = models.User.objects.all()
-    permission_classes = [AnsibleBaseUserPermissions]
+    permission_classes = [OAuth2ScopePermission, AnsibleBaseUserPermissions]
```

Comment:
Should this be wrapped in try_add_oauth2_scope_permission or do we expect services to directly include this permission?

Reply:
I think we can expect services to directly include the permission.